### ISSUE TITLE
(minor) Extra error message logged for Cloud sync in verbose log

### ISSUE THREAD

**Issue by cmitu** [libretro/RetroArch#16641]
## Description

If no cloud sync backends are configured, which is also the default configuration when first starting RetroArch, there's an error from the cloud sync code when running with `-v`/`--verbose` or the log level is set to info/error:
```
...
[ERROR] Couldn't find any cloud sync driver named ""
[INFO] Available cloud sync drivers are:
[INFO] 	webdav
[INFO] 	null
...
```
### Expected behavior

The severity of the message should probably be either Debug or Info when there's no cloud sync backend configured.

### Actual behavior

No error message should be logged, since this is actually a missing configuration not an error. 

### Steps to reproduce the bug

1. Start RetroArch from command line with `retroarch -v`.
2. Note that the error message shows up early in the log output.

### Bisect Results

Not applicable, but probably when the cloud sync code was added. Noticed since now any Linux build enables automatically the cloud sync code ([added](https://github.com/libretro/RetroArch/pull/16456)).

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.19.0 (e79aa14)

### Environment information

- OS: Linux, RaspiOS 12 (aarch64)
- Compiler: version 12.2.0 (Debian 12.2.0-14)

**Comment by warmenhoven** [comment]
Yeah this is because it doesn't set the default cloud sync driver to null, instead it sets it to the empty string which isn't valid. I've been meaning to clean it up.

**Comment by warmenhoven** [comment]
I did finally clean this up.
